src/expandableCalendar/style.js

```diff
@@ -44,15 +44,17 @@ export default function styleConstructor(theme = {}) {
       backgroundColor: '#e8ecf0'
     },
     sectionText: {
-      fontWeight: 'bold', 
-      fontSize: 12,
-      lineHeight: 16,
-      color: '#7a92a5',
-      paddingTop: 24, // 8
-      paddingBottom: 8,
-      paddingLeft: 20,
       backgroundColor: appStyle.calendarBackground,
-      textAlign: 'left'
+      color: '#7a92a5',
+      fontWeight: '500',
+      fontSize: 14,
+      fontFamily: 'Sul Sans',
+      fontStyle: 'normal',
+      letterSpacing: 0,
```

Comment:
ABC

Reply:
Only because it looks like there was an attempt

Reply:
My bad. Thank you.